src/main/starlark/builtins_bzl/common/cc/cc_library.bzl

```diff
@@ -210,7 +210,7 @@ def _cc_library_impl(ctx):
             linker_input = cc_common.create_linker_input(
```

Comment:
I'm guessing the branch condition above (L209) also needs an `if len(ctx.files.additional_linker_inputs) > 0`? (Maybe assigning it to a variable before to save some space)

Reply:
Added the check in cf5c9493ec6c41f75d8686e502498c022509565c.